tools/sof-dump-status.py

```diff
@@ -448,7 +448,15 @@ def dump_dapm(dapm, filter = "all"):
                 pcm['channel'] = '2'
                 pcm['dev'] = 'hw:{},{}'.format(card_id, pcm['id'])
             pipeline_lst.extend(card_info['pcm'])
-        export_pipeline(pipeline_lst)
+        try:
+            pipeline_type = list(filter(lambda x: x.strip().startswith('type'), ret_args['export'].split(' ')))[0].split(':')[1]
+        except:
+            print('Invalid filter string')
```

Comment:
why do we need this filter exactly? What else can the type be?

Reply:
some test case may only want to test on playback pipelines, or capture pipelines. so the type here may be one of the following: playback, capture, any.  As we want to make it compatible with the topology pipeline, so the filter string should be written like this: 'type:playback'